app/views/navbar.scala.html

```diff
@@ -231,6 +231,10 @@ <h4 class="modal-title" id="sign-up-label">Sign up</h4>
                     // Need root directory to prevent cursor icons from disappearing
                     Main({init: "noInit", rootDirectory: "/assets/javascripts/SVLabel/"})
                         .loadData(neighborhood, svl.taskContainer, svl.missionModel, svl.neighborhoodModel);
+
+                    // Prevent first-mission popups
+                    svl.map.unbindPositionUpdate(svl.initialMissionInstruction._instructToCheckSidewalks);
```

Comment:
This line throws an error:

![image](https://user-images.githubusercontent.com/2873216/27978848-011ad2bc-6340-11e7-9e47-4e4d021c4996.png)